### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat '/Users/yanzhang/Documents/CraftAcademy/...

### ISSUE THREAD

**Issue by flamesoft** [tomkadwill/atom-cucumber-autocomplete#16]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.7.4
**System**: Mac OS X 10.11.4
**Thrown From**: [cucumber-autocomplete](https://github.com/tomkadwill/atom-cucumber-autocomplete) package, v0.6.3
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/Users/yanzhang/Documents/CraftAcademy/week6_rails/features'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat '/Users/yanzhang/Documents/CraftAcademy/week6_rails/features'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.scanFeaturesDir (/Users/yanzhang/.atom/packages/cucumber-autocomplete/lib/provider.js:46:13)
    at Object.getCompletions (/Users/yanzhang/.atom/packages/cucumber-autocomplete/lib/provider.js:31:19)
    at Object.getSuggestions (/Users/yanzhang/.atom/packages/cucumber-autocomplete/lib/provider.js:17:17)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:310:56
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:289:17)
    at AutocompleteManager.getSuggestionsFromProviders (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:276:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/autocomplete-plus/lib/autocomplete-manager.js:3:61
```
### Commands

```
     -4:06.9.0 project-find:show (atom-text-editor.editor.is-focused)
     -4:04.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:27 tabs:close-other-tabs (li.texteditor.tab.sortable.active.right-clicked)
     -0:14.3.0 editor:delete-to-beginning-of-line (atom-text-editor.editor.is-focused)
     -0:14 core:backspace (atom-text-editor.editor.is-focused)
     -0:12.7.0 editor:delete-to-beginning-of-line (atom-text-editor.editor.is-focused)
     -0:12.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:08.9.0 editor:delete-to-beginning-of-line (atom-text-editor.editor.is-focused)
     -0:08.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:02.0 editor:newline (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.29.2
autoclose-html, v0.23.0
cucumber, v0.5.0
cucumber-autocomplete, v0.6.3
pigments, v0.26.0

# Dev
No dev packages
```

**Comment by tomkadwill** [comment]
Hi @flamesoft, did you get this working? You can override the default path to your features directory to point to the correct location

**Comment by flamesoft** [comment]
@tomkadwill thanks. I will try that solution.

**Comment by joscha** [comment]
@tomkadwill would you accept a PR that guesses this based on the file where the autocomplete happens?

**Comment by tomkadwill** [comment]
@joscha sure! that would be great

**Comment by tomkadwill** [comment]
Hi @joscha, I've added a fix for this in v0.7.1. Please re-test re-open this issue if the problem still occurs in v0.7.1.

Feel free to open a PR, like you suggested, if you still think it's relevant.
